lib/chat_api_web/channels/notification_channel.ex

```diff
@@ -5,24 +5,19 @@ defmodule ChatApiWeb.NotificationChannel do
   alias ChatApi.{Chat, Conversations}
 
   @impl true
-  def join("notification:lobby", payload, socket) do
-    if authorized?(payload) do
-      {:ok, socket}
+  def join("notification:" <> account_id, %{"ids" => ids}, socket) do
+    if authorized?(socket, account_id) do
```

Comment:
(only allow joining the channel if the authed user's account matches the account id)